Samples/All/BumbleRumble/Common/NetworkManager.cpp

```diff
@@ -485,19 +457,22 @@ void NetworkManager::SendTextMessage(std::string text)
 
 void NetworkManager::SendTextAsVoice(std::string text)
 {
-    if (m_localChatControl != nullptr)
+    if (m_enableCognitiveServices)
     {
-        DEBUGLOG("Requesting transcription of: %hs\n", text.c_str());
+        if (m_localChatControl != nullptr)
+        {
+            DEBUGLOG("Requesting transcription of: %hs\n", text.c_str());

```

Comment:
this comment is not correct?

Reply:
I made no changes to the existing logging. I just moved it inside of the if statement.

Reply:
well, it might be a good time to fix it since it is incorrect.